libraries/apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/introspection/IntrospectionSchema.kt

```diff
@@ -147,51 +153,36 @@ data class IntrospectionSchema(
   }
 }
 
-/**
- *
- */
+@OptIn(ExperimentalSerializationApi::class)
+private val json: Json by lazy {
+  Json {
+    ignoreUnknownKeys = true
+    classDiscriminator = "kind"
+    explicitNulls = false
+  }
+}
+
+@OptIn(ExperimentalSerializationApi::class)
```

Comment:
I don't think it's ok to use this. If a future version changes the signatures, they will break at runtime.

Reply:
But we bundle and relocate the code, so we should be immune to changes? (OTOH I'm ok to avoid it)

Reply:
We relocate in `apollo-gradle-plugin` but some users might use `apollo-ast` outside the Gradle context so I'd feel safer avoid this for the time being.

Reply:
That makes sense 👍